### PR TITLE
Prac 06 Feedback Request

### PR DESCRIPTION
**Pull request by nicknlowe**
@scott-gills-dyer - would you be able to review my code and provide feedback where necessary?

Thanks!

### CODE REVIEW

File: prac_06/guitars.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:
+        print("These are my guitars:")
+        for i, guitar in enumerate(guitars, 1):
+            print(
+                f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f}"
+                f"{' (vintage)' if guitar.is_vintage() else ''}"
```
Comment (scott-gills-dyer) [inline prac_06/guitars.py:31]: You could try Separating the conditional from the print statement to improve readability.
adds a slight maintenance burden if later you would like to add additional categories like "retro" or "antique".

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+"""
+CP1404/CP5632 Practical
+Guitar program that uses the Guitar class
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Guitar program."""
```
Comment (scott-gills-dyer) [inline prac_06/guitars.py:10]: You could add a little more description to this docstring, just what does this guitar program do, such as:
"""Program take gets guitar information and prints formatted list""" or something

**Review by scott-gills-dyer — COMMENTED** [review]
Other than not putting in your times in the docsting for language and guitar programs, everything looks good pretty good no major issues.

File: prac_06/guitars.py (hunk 2 above)
Comment (nicknlowe) [reply prac_06/guitars.py:10]: Sounds good, thank you for your feedback!

**Pull request merged**